src/components/Task.tsx

```diff
@@ -1,20 +1,46 @@
-import { Task as TaskType} from "./TaskInterface";
+import { Task as TaskType } from "./TaskInterface";
 import "./tasks.css";
 
-const Task = (props: TaskType) => {
+export interface TaskProps extends TaskType {
+  markCompleted: (task: string) => void;
+  deleteTask: (task: string) => void;
+}
+
+const Task = (props: TaskProps) => {
+  console.log(props);
   return (
     <div className="task">
       <div className="task-content">
-        <p className={`task-name ${props.isCompleted && "completed"}`}>{props.task}</p>
-        {!props.isCompleted &&  <div className="info">
-          <p className="priority">{props.priority}</p>
-          <p className="category">{props.category}</p>
-        </div>}
+        <p className={`task-name ${props.isCompleted && "completed"}`}>
```

Comment:
You may be interested in this library:
- https://www.npmjs.com/package/classnames

It gives you a good API to do similar things.
With that library, the above could be simplified to:
 ```suggestion
        <p className={classNames('task-name', {'completed': props.isCompleted  })}>
```